README.md

```diff
@@ -23,21 +25,37 @@ Pick your preferred tool to get started:
 
 - For JavaScript and TypeScript developers, install [Teams Toolkit for Visual Studio Code](https://docs.microsoft.com/microsoftteams/platform/toolkit/teams-toolkit-fundamentals#install-teams-toolkit-for-visual-studio-code).
 - For .NET developers, install [Teams Toolkit for Visual Studio](https://docs.microsoft.com/microsoftteams/platform/toolkit/visual-studio-overview#install-teams-toolkit-for-visual-studio).
-- For command line users, install [TeamsFx CLI](https://docs.microsoft.com/microsoftteams/platform/toolkit/teamsfx-cli#get-started): `npm install -g @microsoft/teamsfx-cli`
+- For command line users, install [Teams Toolkit CLI](https://docs.microsoft.com/microsoftteams/platform/toolkit/teamsfx-cli#get-started): `npm install -g @microsoft/teamsfx-cli`
+
+Visit [Microsoft Teams developer documentation](https://aka.ms/teamsfx-docs) to get started with building apps with Teams Toolkit today.
+
+## Road Map
+
+Teams Toolkit for Visual Studio, Visual Studio Code, and Command Line Interface (CLI) will be updated regularly with new features and bug fixes to continuously improve end-to-end Teams development experience. Visit our [product road map](https://aka.ms/teamsfx-roadmap) to find out what's coming.
+
+## Support Policy
+
+Teams Toolkit products will follow [Modern Lifecycle Policy](https://docs.microsoft.com/lifecycle/policies/modern) unless otherwise noted. Visit our [lifecycle and support document](https://aka.ms/teamsfx-support) to find out details for each product.
```

Comment:
can be more accurate as .... will follow Modern Lifecycle Policy and extended support as described in our lifecycle and support document.